CMakeLists.txt

```diff
@@ -34,22 +34,30 @@ endif()
 
 option(SOLC_LINK_STATIC "Link solc executable statically on supported platforms" OFF)
 option(SOLC_STATIC_STDLIBS "Link solc against static versions of libgcc and libstdc++ on supported platforms" OFF)
-option(STRICT_Z3_VERSION "Use the latest version of Z3" ON)
+option(STRICT_Z3_VERSION "Require the exact version of Z3 solver expected by our test suite." ON)
```

Comment:
Maybe we can rename `LATEST_Z3_VERSION` to something like `EXPECTED_Z3_VERSION` then so it stays consistent :)

Reply:
Good idea. But how about `TESTED_Z3_VERSION`? Or `Z3_VERSION_FOR_TESTING`?

Reply:
both your suggestions work just as well for me!